languages/python/exercises/concept/comments/.docs/introduction.md

```diff
@@ -0,0 +1,5 @@
+Comments are part of every programming language. They provide important information about the program to its stakeholders. Every programmer has a different habit of logging comments in the program. Some organizations insist programmers log little detail as possible. So the next programmers would not get confused with the previous person's work.
```

Comment:
Change to "Some organizations insist programmers log as little detail as possible." for grammatical correctness